GuessNumber.c

```diff
@@ -1,5 +1,7 @@
+
 #include <stdio.h>
 #include <time.h>
+#include <windows.h>
```

Comment:
@A-Arellano have you included `windows.h` for `system(cls)` system call? Are you not running the program on a Linux based distro or on GCC/G++ (GNU based compilers), you may want to run on DevC++ if you are working on Windows environment.